app/src/main/java/com/mapbox/maps/testapp/examples/MapViewCustomizationActivity.kt

```diff
@@ -21,11 +21,10 @@ class MapViewCustomizationActivity : AppCompatActivity() {
 
   override fun onCreate(savedInstanceState: Bundle?) {
     super.onCreate(savedInstanceState)
-    // Create  a custom CredentialsManager with a token and set it to CredentialsManager.shared, so that all MapViews created with default config will apply this token.
-    CredentialsManager.default.setAccessToken(getString(R.string.mapbox_access_token))
 
-    // Create a custom ResourceOptionsManager with customised tile store and tile size, so that all MapViews created with default config will apply these settings.
+    // Create a custom ResourceOptionsManager with customised token, tile store and tile size, so that all MapViews created with default config will apply these settings.
     ResourceOptionsManager.getDefault(this).update {
+      accessToken(getString(R.string.mapbox_access_token))
```

Comment:
So do I get it correct that this token will be used by default for all other `MapView`s created afterwards?

Reply:
That's right.

Reply:
@Chaoba here will throw exceptions if the default accesstoken does not exist in Android resources, probably we should move the setting of the access token to the ResourceOptionsManager.getDefault(this, getString(R.string.mapbox_access_token))